Gemfile

```diff
@@ -72,6 +72,7 @@ group :test do
   gem 'rspec'
   gem 'rspec-hanami', github: 'davydovanton/rspec-hanami'
   gem 'capybara'
+  gem 'capybara-screenshot'
```

Comment:
Could you explain why we need this gem? :)

Reply:
@davydovanton It's simplify debugging for feature tests